### PR TITLE
Fix bindir and executables in gemspec

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

**Comment by ShimShtein** [comment]
Fixes https://github.com/ShimShtein/detective/issues/2. 
@ekohl, would you like to ACK?

File: detective.gemspec [hunk 1]
```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |spec|
   spec.files = `git ls-files -z`.split("\x0").reject do |f|
     f.match(%r{^(test|spec|features)/})
   end
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = 'bin'
+  spec.executables   = %w[console detective setup]
```
Comment (ekohl) [inline detective.gemspec]: Should console and setup really be part of the gem? I also think I tried this when I wanted to write a patch but when I set ```bindir``` to ```bin``` and would only name ```detective``` it'd still include both the other bins so this line might even be redundant *if* you want to include them.

**Comment by ShimShtein** [comment]
@ekohl, you are right!
Removed the setup and console bins.

**Review by ekohl — APPROVED** [review]

**Comment by ShimShtein** [comment]
Merged!
Thanks @ekohl for the revew!

**Pull request merged**